<launch>
  <arg name="ns" default="myrobot"/>
  <arg name="id_subsystem" default="133"/>
  <arg name="id_node" default="64"/>
  <group ns="$(arg ns)">
      <node name="iop_costmap2d" pkg="fkie_iop_component" type="iop_component">
        <param name="capability_group" value="iop_bridge"/>
        <rosparam param="services">
          [
            fkie_iop_costmap2d: "urn:jaus:jss:iop:CostMap2D",
            fkie_iop_discovery: "urn:jaus:jss:core:DiscoveryClient",
          ]
        </rosparam>
        <param name="iop_address" value="$(arg id_subsystem).$(arg id_node).71"/>
        <param name="enable_ros_interface" value="false"/>
        <param name="register_own_services" value="true" />
        <param name="offset_yaw" value="1.5707" />
        <param name="robot_frame" value="base_link" />
    </node>
  </group>
</launch>
